instrumentserver/server/core.py

```diff
@@ -695,12 +695,12 @@ def _callObject(self, spec: CallSpec) -> Any:
                 self.parameterSet.emit(spec.target, args[0])
 
                 # broadcast changes in parameter values
-                self._broadcastParameterChange(ParameterBroadcastBluePrint(spec.target, 'set', args[0]))
+                self._broadcastParameterChange(ParameterBroadcastBluePrint(spec.target, 'parameter-update', args[0]))
```

Comment:
is it possible for this to fail when someone passes the value as a keyword? please double check.

Reply:
I am not entirely sure what you mean by passing values as keyword. I have tried typing an int directly instead of a variable, a None and True and False and they all work (the parameter manager gui displays text then). It behaves the same way if you pass a string.